pkg/util/ecs/metadata/v3or4/client_test.go

```diff
@@ -126,3 +127,7 @@ var expected = &Task{
 		},
 	},
 }
+
+func toPointer[T any](v T) *T {
```

Comment:
Such a function already exists: https://github.com/DataDog/datadog-agent/blob/7b965903fbb5ebaa21a9d9ddea10ba23265826ee/pkg/util/pointer/pointer.go#L9-L12

Reply:
thanks, I updated the PR